{{cookiecutter.project_slug}}/clients/vue3/src/components/inputs/InputField.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <div class="form-block">
-    <label v-if="label" :for="`${label}-field`" v-text="label" />
+  <div class="input-field flex flex-col items-start mb-2 w-full">
```

Comment:
**Info:** Updated the descriptive class name to be `input-field` to match the naming of the component (`InputField`) for clarity. It is now also not being used for style targeting.